### DISCUSSION TITLE
Invalid Remainder

### DISCUSSION THREAD

**Discussion by fzavalia** [0xPolygon/matic.js#319] [Q&A] [answered]
Hey!

A user reported the following error while trying to finish a withdraw.

![image](https://user-images.githubusercontent.com/24811313/153936905-fa89f975-ff37-478b-9267-f803f4990e1b.png)

We are currently using what seems an outdated version of the package (2.0.34) while the latest at the moment is 3.2.2.

We are planning on upgrading, yet I would like to know if this issue is cause by the version or something else.

(Only one user has reported this issue)

**Comment by ujjwalgupta94** [comment, marked as answer]
We recommend to use the v3 of maticjs. But for now - you can upgrade to `2.0.51` version and the issue will be solved without any code changes from your side.

```
npm i @maticnetwork/maticjs@2.0.51
```

**Reply by fzavalia** [reply]
Thank you!
